dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/deduplication/DeduplicationController.java

```diff
@@ -97,40 +92,37 @@
 
     private final FieldFilterService fieldFilterService;
 
-    private final ContextService contextService;
+    private static final String DEFAULT_FIELDS_PARAM = "created, lastUpdated, original, duplicate, status";
 
     @OpenApi.Response( PotentialDuplicate[].class )
     @GetMapping
-    public Node getAllByQuery(
-        PotentialDuplicateQuery query,
-        HttpServletResponse response )
+    public PagingWrapper<ObjectNode> getPotentialDuplicates(
+        PotentialDuplicateCriteria potentialDuplicateCriteria,
+        HttpServletResponse response,
+        @RequestParam( defaultValue = DEFAULT_FIELDS_PARAM ) List<String> fields )
         throws BadRequestException
     {
-        checkDeduplicationStatusRequestParam(
-            Optional.ofNullable( query.getStatus() ).map( Enum::name ).orElse( null ) );
+        checkDeduplicationStatusRequestParam( potentialDuplicateCriteria.getStatus() );
```

Comment:
This check will never throw an Exception. If an invalid value is passed for the DeduplicationStatus then Spring will throw a `BindingException` that should be intercepted in CrudControllerAdvice and translated to a BadRequestException. The interceptor for this exception is not implemented yet